content/appendices.md

```diff
@@ -940,6 +940,30 @@ The acceptable character set matches the unreserved character set in [RFC
 3986](https://datatracker.ietf.org/doc/html/rfc3986#section-2.3).
 {{% /boxes/note %}}
 
+## Cryptographic key representation
+
+Sometimes it is necessary to present a private cryptographic key in the user
+interface.
+
+When this happens, the key should be presented as a string formatted as
+follows:
+
+1.  The raw key is prepended by the two bytes `0x8B` and `0x01`.
```

Comment:
This should probably clarify if the key itself is the initial array.

Reply:
I did some rewording